index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+
+
+</head>
+<body>
+    <nav class="navigation">
+       <img src="./logos/MyLogo.png" alt="navigation icon" class="navigation__logo">
+       <img src="./logos/menu.svg" alt="navigation icon" class="navigation__hamburger">
+    </nav>
+    
+<div class="bg">
+    <h1>I’m Mariane
+        Glad to see you!</h1>
```

Comment:
- [ ] You should use different heading tags in for the two texts here. You main text is  `I'm Mariane` then you have a subtopic of `Glad to see you!` Thus, you should `<h1>` to main topic and `<h2>` to the next sub-heading. **NB**: we don't use heading tags for font-size and font-weight purposes, but for their semantic meaning.